epam-promo/components/fileUpload/FileCard.tsx

```diff
@@ -14,20 +14,25 @@ import { ReactComponent as TableIcon } from '../../icons/fileUpload/file-file_ta
 import { ReactComponent as TextIcon } from '../../icons/fileUpload/file-file_text-24.svg';
 import { ReactComponent as MailIcon } from '../../icons/fileUpload/file-file_eml-24.svg';
 import { ReactComponent as FileIcon } from '../../icons/fileUpload/file-file-24.svg';
+import { ReactComponent as ErrorIcon } from '../../icons/notification-error-fill-10.svg';
+
+interface NewFile extends Partial<File> {
+    id?: number;
+    progress?: number;
+    abortXHR?: () => void;
+    uploadError?: { isError: boolean, message?: string };
```

Comment:
info about error will be extended from File interface, we don't need to define it here one more time